models/marts/core/dim_customers.sql

```diff
@@ -0,0 +1,40 @@
+with customers as (
+    select * from {{ ref('stg_customers')}}
+),
+orders as (
+    select * from {{ ref('stg_orders')}}
+),
+customer_orders as (
+    select
+        customer_id,
+        min(order_date) as first_order_date,
+        max(order_date) as most_recent_order_date,
+        count(order_id) as number_of_orders,
+        sum(amount) as lifetime_value_total
+    from analytics.analytics.orders

```

Comment:
>    from analytics.analytics.orders

You actually want `orders` here because of the CTE above where you have defined what `orders` is. Also our rule of thumb is to never hardcode any models ( so analytics.analytics.orders).

Reply:
makes sense but I switched that explicit table because the CTE above for orders didnt have amount and was throwing an error, Ill look into stg_orders to see how I can add amount field there to funnel through. thanks for the advice